Firmware/RTK_Surveyor/AP-Config/src/main.js

```diff
@@ -303,13 +303,15 @@ function parseIncoming(msg) {
         else if (val == "true") {
             try {
                 ge(id).checked = true;
+                console.log("Set " + id + ": " + val);
             } catch (error) {
                 console.log("Issue with ID: " + id)
             }
         }
         else if (val == "false") {
             try {
                 ge(id).checked = false;
+                console.log("Set " + id + ": " + val);
```

Comment:
This is fine for your own local debugging, I am hesitant to console log anything but errors, just because I don't want to load the user's browser. Would you please comment these out?